plasmapy/analysis/time_series/tests/test_excess_statistics.py

```diff
@@ -0,0 +1,91 @@
+"""Tests for excess_statistics.py"""
+
+import astropy.units as u
+import numpy as np
+import pytest
+
+from plasmapy.analysis.time_series.excess_statistics import ExcessStatistics
+
+
+@pytest.mark.parametrize(
+    "signal, thresholds, time_step, pdf, bins, expected",
+    [
+        (
+            [0, 2, 0],
+            1,
+            1,
+            False,
+            32,
+            ([1], [1], [1], [0]),
+        ),
+        (
+            [0, 2, 0],
+            1.5,
+            1,
+            False,
+            32,
+            ([1], [1], [1], [0]),
+        ),
+        (
+            [0, 2, 2, 0, 4, 0],
+            [1, 3, 5],
+            1,
+            False,
+            32,
+            ([3, 1, 0], [2, 1, 0], [1.5, 1, 0], [0.5, 0, 0]),
+        ),
+        (
+            np.array([2, 2, 0, 4, 0]),
+            [1, 3, 5],
+            1,
+            False,
+            32,
+            ([3, 1, 0], [1, 1, 0], [1.5, 1, 0], [0.5, 0, 0]),
+        ),
+        (
+            [0, 2, 0, 4, 0] * u.eV,
+            [1, 3, 5],
+            1,
+            False,
+            32,
+            ([2, 1, 0], [2, 1, 0], [1, 1, 0], [0, 0, 0]),
+        ),
+        (
+            [0, 2, 0, 4, 0],
+            [1, 3, 5],
+            1,
+            True,
+            2,
+            (
+                [2, 1, 0],
+                [2, 1, 0],
+                [1, 1, 0],
+                [0, 0, 0],
+                [[0, 2], [0, 2], [0, 0]],
+                [[0.75, 1.25], [0.75, 1.25], [0, 0]],
+            ),
+        ),
+    ],
+)
+def test_ExcessStatistics(signal, thresholds, time_step, pdf, bins, expected):
+    """test excess_stat function"""
+    tmp = ExcessStatistics(signal, thresholds, time_step)
+    assert tmp.total_time_above_threshold == expected[0]
+    assert tmp.number_of_crossings == expected[1]
+    assert tmp.average_times == expected[2]
+    assert tmp.rms_times == expected[3]
+    if pdf:
+        hist, bin_centers = tmp.hist(bins)
```

Comment:
```suggestion
    excess_stats = ExcessStatistics(signal, thresholds, time_step)
    assert excess_stats.total_time_above_threshold == expected[0]
    assert excess_stats.number_of_crossings == expected[1]
    assert excess_stats.average_times == expected[2]
    assert excess_stats.rms_times == expected[3]
    if pdf:
        hist, bin_centers = excess_stats.hist(bins)
```
I'm wondering if a more explicit name would help improve readability.